web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/fordeling/FordelRestTjeneste.java

```diff
@@ -324,13 +324,17 @@ private InfoOmSakInntektsmeldingResponse mapInfoOmSakInntektsmelding(Behandling
             .map(Aksjonspunkt::getAksjonspunktDefinisjon)
             .collect(Collectors.toSet());
 
-        var førsteUttaksdato = skjæringstidspunktTjeneste.getSkjæringstidspunkter(behandling.getId()).getFørsteUttaksdatoHvisFinnes();
+
+        var skjæringstidspunkter = skjæringstidspunktTjeneste.getSkjæringstidspunkter(behandling.getId());
+        var førsteUttaksdato = skjæringstidspunkter.getFørsteUttaksdatoHvisFinnes();
+
         var inntektsmeldingStatusSak = mapInntektsmeldingStatusSak(aksjonspunkterIkkeKlarForInntektsmelding);
         if (inntektsmeldingStatusSak.equals(StatusSakInntektsmelding.ÅPEN_FOR_BEHANDLING) && førsteUttaksdato.isEmpty()) {
             throw new IllegalStateException("Ulovlig tilstand-infoOmSakForInntektsmelding: Finner ikke førsteUttaksdato for behandling " + behandling.getId() + "med inntektsmeldingsstatus ÅPEN_FOR_BEHANDLING");
         }
 
-        return new InfoOmSakInntektsmeldingResponse(inntektsmeldingStatusSak, førsteUttaksdato.orElse(Tid.TIDENES_ENDE));
+        return new InfoOmSakInntektsmeldingResponse(inntektsmeldingStatusSak, førsteUttaksdato.orElse(Tid.TIDENES_ENDE),
```

Comment:
Burde vi feile hvis vi ikke finner stp? Vet ikke helt hva slags scenario det kan være, men vet ikke helt hvordan det blir hvis fpinntektsmelding begynner å slå opp inntekter 3 mnd før 31.12.9999

Reply:
Jeg er ukjent med prosessen rundt disse med tomt uttak. Enten skjer det i en revurdering vil jeg anta, og da er det allerede bedt om inntektsmelding. Da mener jeg vi har regler som utleder når ny søknad kommer om det er behov for å innhente ny inntektsmelding? 
Vi er jo på unntaktstilfeller her - altså agi innsendte inntektsmeldinger, så vi må antagelig lære litt underveis.